spec/helpers/textual_mixins/textual_availability_spec.rb

```diff
@@ -1,22 +0,0 @@
-describe TextualMixins::TextualAvailability do
-  describe '#translated_status' do
```

Comment:
the status translation is only useful for middleware?

Reply:
Yes, not used anywhere else.